packages/amplify-provider-awscloudformation/src/aws-utils/aws-lex.js

```diff
@@ -18,6 +18,7 @@ const serviceRegionMap = {
   'ap-northeast-1': 'ap-northeast-1',
   'eu-central-1': 'eu-central-1',
   'eu-north-1': 'eu-central-1',
+  'eu-south-1': 'eu-central-1',
```

Comment:
Lex is not available in eu-south-1. Falling back to eu-central-1.

https://docs.aws.amazon.com/general/latest/gr/lex.html